packages/runtime-logger/src/internalPlugin.ts

```diff
@@ -1,8 +1,11 @@
 import type { Plugin } from 'vite';
+import { readFile } from 'fs/promises';
 
 export const INTERNAL_MODULE = '@it-astro:logger-internal';
 const RESOLVED_INTERNAL_MODULE = '\x00@it-astro:logger-internal';
 
+const templatePath = new URL('../template/loggerStub.mjs', import.meta.url);
```

Comment:
Don't forget to add `template` to `files` in the package.json if it's not the case already!

Reply:
Thank you!